config/ui_bash_profile

```diff
@@ -1,7 +1,7 @@
 export PATH=/vx/code/config/admin-functions:${PATH}
 
 if [[ $(tty) = /dev/tty1 ]]; then
-   exec startx
+   XAUTHORITY=/tmp/.Xauthority startx
```

Comment:
This no longer replaces the current process with `startx`, so if `startx` exits it'll continue the script. Is that intended?

Reply:
Intended yes, but not as a final end-state, I'm playing with a few other things.

Are we certain that with `exec` it will cause a logout if you ctrl-C while it boots?

Reply:
I'm not sure how `ui_bash_profile` is run, so I'm not sure about its behavior with regard to logouts. I'm just saying that this will print `foo`:

```sh
run_something_that_exits
echo foo
```

and this will not:

```sh
exec run_something_that_exits
echo foo
```